test/test_commands.py

```diff
@@ -59,6 +59,76 @@ class TestCommand(AlignakTest):
     This class tests the commands
     """
 
+    def setUp(self):
+        self.setup_with_file('cfg/cfg_commands.cfg')
+        assert self.conf_is_correct
+
+        # Our scheduler
+        self._sched = self.schedulers['scheduler-master'].sched
+
+    def test_css_in_commands(self):
```

Comment:
Ok, but add a comment in the function description please to be enough explicit for the user read the file later ;)